### PR TITLE
Add minimum_must_match to search queries

### PR DESCRIPTION
**Pull request by DanielHeath**

### CODE REVIEW

**Comment by DanielHeath** [comment]
Not that it'll do much until ISS supports it.

File: src/iss.js [hunk 1]
```diff
@@ -394,6 +396,7 @@ export async function search(
     let request_: searchRequest = {
         q: "",
         type: "service",
+        minimum_must_match: "3>50%",
```
Comment (kane-c) [inline src/iss.js]: I'm not sure if `>` is allowed. Docs suggest `<` or `<-`
https://www.elastic.co/guide/en/elasticsearch/reference/1.4/query-dsl-minimum-should-match.html

**Comment by DanielHeath** [comment]
Oh right - sign is backwards. For `50%` I think `<` and `<-` are equivalent, right?

**Comment by DanielHeath** [comment]
@kane-c ++?

**Comment by kane-c** [comment]
++

**Pull request merged**